osmomath/decimal_test.go

```diff
@@ -38,6 +38,59 @@ func (s *decimalTestSuite) assertMutResult(expectedResult, startValue, mutativeR
 	s.Require().Equal(nonMutativeStartValue, startValue)
 }
 
+func TestAddMut(t *testing.T) {
```

Comment:
Since `Add` is supposed to call `AddMut`, can you please do a short test [like this](https://github.com/osmosis-labs/osmosis/blob/62067092a726b59117345e11f3355e381a72bf3c/osmomath/decimal_test.go#L1184-L1220) instead of duplicating test cases?